unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/PointerEvents/PointerEventsHandler.cs

```diff
@@ -0,0 +1,45 @@
+﻿using DCL.Controllers;
+using DCL.ECS7.InternalComponents;
+using DCL.ECSRuntime;
+using DCL.Models;
+
+namespace DCL.ECSComponents
+{
+    public class PointerEventsHandler : IECSComponentHandler<PBPointerEvents>
+    {
+        private readonly IInternalECSComponent<InternalPointerEvents> internalPointerEvents;
+
+        public PointerEventsHandler(IInternalECSComponent<InternalPointerEvents> internalPointerEvents)
+        {
+            this.internalPointerEvents = internalPointerEvents;
+        }
+
+        public void OnComponentCreated(IParcelScene scene, IDCLEntity entity) { }
+
+        public void OnComponentRemoved(IParcelScene scene, IDCLEntity entity)
+        {
+            internalPointerEvents.RemoveFor(scene, entity);
+        }
+
+        public void OnComponentModelUpdated(IParcelScene scene, IDCLEntity entity, PBPointerEvents model)
+        {
+            InternalPointerEvents internalPointerEventsModel = new InternalPointerEvents();
```

Comment:
Couldn't this model be created on `OnComponentCreated()` and then manipulate that model instance in the `OnComponentModelUpdated()` instead of creating a new one every time?